documentation/book/assembly-access-configuration-kafka-mirror-maker.adoc

```diff
@@ -0,0 +1,18 @@
+// This assembly is included in the following assemblies:
+//
+// assembly-deployment-configuration.adoc
+
+[id='assembly-access-configuration-kafka-mirror-maker-{context}']
+= Configuring Kafka Mirror Maker authentication
+
+Configure Kafka Mirror Maker authentication using supported mechanisms:
+
+* TLS encryption
```

Comment:
if we talk about authentication, putting "TLS encryption" doesn't make sense. It's not an authentication method. It's a way to secure connection which then provides client authentication which is the next item. I would remove this one.

Reply:
@ppatierno -- Do you want the TLS encryption section removed to its own section or can we just change the title to 'Configuring Kafka Mirror Maker encryption and authentication'?

Reply:
What's the purpose of this paragraph then? Talking about encryption or authentication? They are two different things. The thing that can confuse is that TLS client authentication is a type of authentication based on TLS which is for encryption. But again encryption and authentication are two different things. Removing TLS encryption, or changing the title ... all depends on what you had in mind about this paragraph.

Reply:
![image](https://user-images.githubusercontent.com/47596553/64701444-b4854700-d4a0-11e9-9164-c70e696c4cf6.png)